lib/node_modules/@stdlib/stats/base/ndarray/dmeanstdev/lib/main.js

```diff
@@ -0,0 +1,58 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var numelDimension = require( '@stdlib/ndarray/base/numel-dimension' );
+var getStride = require( '@stdlib/ndarray/base/stride' );
+var getOffset = require( '@stdlib/ndarray/base/offset' );
+var getData = require( '@stdlib/ndarray/base/data-buffer' );
+var strided = require( '@stdlib/stats/strided/dmeanstdev' ).ndarray;
+
+
+// MAIN //
+
+/**
+* Computes the mean and standard deviation of a one-dimensional double-precision floating-point ndarray.
+*
+* @param {ArrayLikeObject<Object>} arrays - array-like object containing: input ndarray and output ndarray
+* @returns {Float64Array} output ndarray containing [ mean, stdev ]
+*
+* @example
+* var Float64Array = require( '@stdlib/array/float64' );
+* var ndarray = require( '@stdlib/ndarray/base/ctor' );
+*
+* var xbuf = new Float64Array( [ 2.0, 1.0, 2.0, -2.0, -2.0, 2.0, 3.0, 4.0 ] );
+* var x = new ndarray( 'float64', xbuf, [ 4 ], [ 2 ], 1, 'row-major' );
+* var out = new ndarray( 'float64', new Float64Array( 2 ), [ 2 ], [ 1 ], 0, 'row-major' );
+*
+* var v = dmeanstdev( [ x, out ] );
+* // returns <Float64Array>[ 1.25, 2.5 ]
+*/
+function dmeanstdev( arrays ) {
+	var out = arrays[ 1 ];
+	var x = arrays[ 0 ];
+	return strided( numelDimension( x, 0 ), 1, getData( x ), getStride( x, 0 ), getOffset( x ), getData( out ), getStride( out, 0 ), getOffset( out ) ); // eslint-disable-line max-len
```

Comment:
This is not what is desired. For a much more complicated API, see https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/stats/base/ndarray/dztest. Notice there that scalar parameters, such as should be done for `correction`, are provided as ancillary arrays. Hardcoding `1` for the correction is not what we want.